server/mturk.js

```diff
@@ -1,68 +1,18 @@
-const mturk = Npm.require("mturk-api");
-const JSPath = Npm.require("jspath");
-
-let api = undefined;
-
 if (!TurkServer.config.mturk.accessKeyId || !TurkServer.config.mturk.secretAccessKey) {
   Meteor._debug("Missing Amazon API keys for connecting to MTurk. Please configure.");
 } else {
-  const config = {
-    access: TurkServer.config.mturk.accessKeyId,
-    secret: TurkServer.config.mturk.secretAccessKey,
-    sandbox: TurkServer.config.mturk.sandbox
+  AWS.config = {
+    accessKeyId: TurkServer.config.mturk.accessKeyId,
+    secretAccessKey: TurkServer.config.mturk.secretAccessKey,
+    region: "us-east-1",
+    sslEnabled: "true"
   };
 
-  const promise = mturk
-    .connect(config)
-    .then(api => api)
-    .catch(console.error);
-  api = Promise.resolve(promise).await();
-}
-
-TurkServer.mturk = function(op, params) {
-  if (!api) {
-    console.log("Ignoring operation " + op + " because MTurk is not configured.");
-    return;
-  }
-
-  const promise = api.req(op, params).then(resp => resp);
-  const result = Promise.resolve(promise).await();
-
-  return transform(op, result);
-};
-
-/*
-  Translate results to be a little more similar to the original code:
-  https://github.com/jefftimesten/mturk/blob/master/index.js
-
-  Docs at https://github.com/dfilatov/jspath:
-  expressions always return an array;
-  with [0] at the end return the first match.
-
-  XXX we may not necessarily want to continue using these in the future.
-   This is just for compatibility with what the previous API returned.
- */
-function transform(op, result) {
-  switch (op) {
-    case "CreateHIT":
-      return JSPath.apply("..HITId[0]", result);
-    case "GetAccountBalance":
```

Comment:
The old AWS SDK used to be all XML-based, which means we have this ugly parser stuff. We can remove all that now and access the JSON data directly, but note that this will have to be added to each `TurkServer.mturk` call as in the example code above.